openpype/modules/deadline/plugins/publish/collect_pools.py

```diff
@@ -36,12 +36,21 @@ def process(self, instance):
             instance.data["primaryPool"] = (
                 attr_values.get("primaryPool") or self.primary_pool or "none"
             )
+        if instance.data["primaryPool"] == "-":
+            instance.data["primaryPool"] = None
 
         if not instance.data.get("secondaryPool"):
             instance.data["secondaryPool"] = (
                 attr_values.get("secondaryPool") or self.secondary_pool or "none"  # noqa
             )
 
+        if instance.data["secondaryPool"] == "-":
+            instance.data["secondaryPool"] = None
+
+        self.log.info("prima::{}".format(instance.data["primaryPool"]))
```

Comment:
These should not be info logs. Also please log the full name 'primaryPool' if you do end up logging as debug.

Reply:
@mkolar can we somehow enforce PR reviews to include reviewing debugging levels now with the artist facing reports in the publisher.